public/index.html

```diff
@@ -14,8 +16,8 @@
 </head>
 
 <body>
-  <div id="main" class="loading">
-    <header>
+  <main id="main" class="loading">
+    <header class="header">
```

Comment:
Does assigning a header class here have any benefit?

Reply:
The styling of this element here is shared with the `<h1>` wrapping element in the Order Summary `<aside>` to create the visual illusion of one full width header element. 

This was previously also a `<header>` element but I switched it to a `<div>` and gave them both a `.header` class so they would continue to share the same styling. I switched it to a `<div>` as having them both as header elements mean they both have an implicit `role="banner"` whereas a page should only have one element with a banner role. 

The alternate way is to leave the `<header>` without a class and duplicate the selectors in the CSS like so;

```
.header,
header {
// ...styles
}
```

...but I figure this approach is cleaner.

Reply:
Ah yes, I missed that the aside also had a header element. 
LGTM
Thanks again for submitting this!

Reply:
Pleasure - loving everything Stripe! 👌🏻